runtime/query.go

```diff
@@ -43,13 +43,16 @@ func populateFieldValueFromPath(msg proto.Message, fieldPath []string, values []
 	var props *proto.Properties
 	m = m.Elem()
 	for i, fieldName := range fieldPath {
-		isLast := i == len(fieldPath)-1
+		var isLast = i == len(fieldPath)-1
```

Comment:
Considering making a commit to revert this particular line. It got changed as part of hacking on this a bit (before fieldByProtoName returned an error). I could also amend the commit and submit a new pull request. Does it seem worthwhile to do either?